html-boilerplate/css/index.html

```diff
@@ -7,38 +7,39 @@
 		  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Audiowide|Sofia|Trirong">
 	</head>
 	<body>
-	<section>
-	<div>  
+		<div class="navbar">
+			<a href="#about">About</a>
+			<a href="#experience">Experience</a>
+			<a href="#skills">Skills</a>
+			<a href="#connect">Connect</a>
+			<a href="messages">Messages</a>  
```

Comment:
Just forgot to put the `#` in here, that's why the link wasn't working
```suggestion
			<a href="#messages">Messages</a>  
```